ngmt/utils/preprocessing.py

```diff
@@ -1587,3 +1591,70 @@ def process_postural_transitions_stationary_periods(
 
     # Return the necessary outputs
     return time_pt, pt_type, pt_angle, duration, flexion_max_vel, extension_max_vel
+
+
+# Function to plot results of the turn detection algorithm
+def pham_turn_plot_results(accel, gyro, detected_turns, sampling_freq_Hz):
```

Comment:
For me, plotting the results of gait detection or turn detection should not be part of `preprocessing.py`. Should we discuss this again? I am much more a fan of having something like:
```
from ngmt.viz_utils import plot_turns, plot_gait_sequences, ...

```

Reply:
You are right. I move all plot options to ngmt.viz_utils.